src/stepper/advance.f90

```diff
@@ -23,7 +23,8 @@ module advance_mod
                                        , NC_OXMEAN, NC_OYMEAN, NC_OZMEAN    &
                                        , NC_GMAX, NC_RGMAX, NC_RBFMAX       &
                                        , NC_BFMAX, NC_UMAX, NC_VMAX         &
-                                       , NC_WMAX, NC_USZRMS, NC_USSRMS
+                                       , NC_WMAX, NC_USZRMS, NC_USSRMS      &
```

Comment:
Do we want SRMS anywhere, or just remove?

Reply:
SRMS = rms of strain for the full 3D field?

Reply:
I removed the rms diagnostics.